documentation/translations/translating.rst

```diff
@@ -108,33 +111,173 @@ details.
 `#translations channel <https://discord.gg/h3qDwgyzga>`_, `translation
-mailing list <translation_ml_>`_, and there's a `Libera.Chat IRC
-<https://libera.chat/>`_ channel, ``#python-doc``.
+mailing list <translation_ml_>`_, and the `translations subsection <_discourse>`_
+of the Python Discourse.
+
+
+Style Guide
+===========
+
+Before translating, you should familiarize yourself with the general
+documentation :doc:`style guide<../style-guide>`. Some translation-specific
+guidelines are explained below.
+
+
+Translate, don't rewrite
+------------------------
+
+Try to stay as close as possible to the original text. Focus on translating its
+meaning rather than rephrasing or rewriting it.
+
+
+Gender neutrality
+-----------------
+
+Many languages use grammatical gender. When possible and natural, prefer
+gender-neutral or inclusive forms. Aim to reflect the inclusive tone of
+the English documentation.
+
+
+Roles and links
+---------------
+
+The Python docs contain many roles (``:role:`target```)
+to other parts of the documentation.
+Leave reStructuredText roles such as ``:func:`print``` or ``:ref:`some-section``` in
+place, even if they contain section titles, because it will break the link.
+If alternate text (``:role:`text <target>``` is provided, it can be translated.
+
+Links (```text <target>`_``) should be handled similarly. If possible, the target
+should be updated to match the language.
+
+.. seealso::
+   :doc:`../markup`
+
+
+Code examples
+-------------
+
+Translate values in code examples, that is string literals, and comments.
+Don't translate keywords or names, including variable, function, class, argument,
+and attribute names. An example of a translated codeblock from the `tutorial <https://docs.python.org/3/tutorial/controlflow.html#keyword-arguments>`_
+is provided below:
+
+.. code-block:: python
+
+   def cheeseshop(kind, *arguments, **keywords):
+       print("-- Czy jest może", kind, "?")
+       print("-- Przykro mi, nie mamy już sera", kind)
+       for arg in arguments:
+           print(arg)
+       print("-" * 40)
+       for kw in keywords:
+           print(kw, ":", keywords[kw])
+
+
+Translation quality
+-------------------
+
+Translators should be proficient in both English and the language they are
+translating to. Translators should aim for a similar level of quality as that
+of the English documentation.
+
+Avoid relying solely on machine translation. These tools can be useful to speed up
+work, but often produce inaccurate or misleading results and should be reviewed
+by a human.
+
+
+Terminology
+-----------
+
+The documentation is full of technical terms, some are common in general
+programming and have translations, whereas others are specific to Python
+and previous translations are not available (remember to check your languages
+glossary). Translation teams should keep the translations of these terms
+consistent, which is done with glossaries and or translation memory.
```

Comment:
```suggestion
The documentation is full of technical terms, some are common in general
programming and have translations, whereas others are specific to Python
and previous translations are not available.
Translation teams should keep the translations of these terms
consistent, which is done with glossaries.
```
I think the translation memory has marginal use for that, the glossaries are definitely the solution for consistent terminology, used in both offline and online tools.